src/setupWorker/start/createStart.ts

```diff
@@ -124,6 +124,11 @@ If this message still persists after updating, please report an issue: https://g
         return null
       }
 
+      context.keepAliveInterval = setInterval(
+        () => worker.postMessage('PING'),
+        5000,
```

Comment:
If the kill timeout is rather large for the Service Worker, what are the repercussions of raising this interval?

Reply:
No repercussions - what I'm going to do is make this configurable with sensible defaults so someone could change the timeout to be lower or higher. 30s is supposed to be the time before the worker is stopped, but that behavior seems very random to me and changes by available resources on a machine.